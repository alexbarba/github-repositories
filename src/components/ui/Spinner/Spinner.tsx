const Spinner = ({ className = '' }) => (
  <div className="flex items-center justify-center w-full h-full">
    <div
      className={`border-t-4 border-b-4 border-black rounded-full animate-spin w-32 h-32 dark:border-white ${className}`}
    />
  </div>
)

export default Spinner
