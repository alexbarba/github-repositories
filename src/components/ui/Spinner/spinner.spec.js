import { render } from '@testing-library/react'
import Spinner from './Spinner'

describe('Spinner', () => {
  it('should be match with snapshot', () => {
    const { asFragment } = render(<Spinner />)
    expect(asFragment()).toMatchSnapshot()
  })
})
