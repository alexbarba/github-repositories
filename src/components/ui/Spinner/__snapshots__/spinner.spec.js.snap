// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should be match with snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex items-center justify-center w-full h-full"
  >
    <div
      class="border-t-4 border-b-4 border-black rounded-full animate-spin w-32 h-32 dark:border-white "
    />
  </div>
</DocumentFragment>
`;
